index.js

```diff
@@ -238,22 +238,29 @@ ShareDbMongo.prototype.close = function(callback) {
 
 ShareDbMongo.prototype.commit = function(collectionName, id, op, snapshot, options, callback) {
   var self = this;
-  var request = createRequestForMiddleware(options, collectionName, op);
+  var request = createRequestForMiddleware(null, options, collectionName, op);
```

Comment:
Did you mean to add this `null`? According to the function signature below, `options` should still be the first argument?

Reply:
Yeah the signature changed a couple times in the PR, will update...

Reply:
Yeah the signature changed a couple times in the PR, will update...